support-frontend/stories/checkoutLayout/CheckoutHeading.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import { brand } from '@guardian/source-foundations';
+import { Column, Columns } from '@guardian/source-react-components';
+import type { CheckoutHeadingProps } from 'components/checkoutHeading/checkoutHeading';
+import { CheckoutHeading } from 'components/checkoutHeading/checkoutHeading';
+import { Container } from 'components/layout/container';
+import { withPositionRelative } from '../../.storybook/decorators/withPositionRelative';
+
+export default {
+	title: 'Checkout Layout/Checkout Heading',
+	component: CheckoutHeading,
+	argTypes: {
+		image: {
+			table: {
+				disable: true,
+			},
+		},
+	},
+	decorators: [withPositionRelative],
+	parameters: {
+		docs: {
+			description: {
+				component:
+					'A heading block that sits behind part of the UI. It must be placed inside a relative positioned container. A subsequent sibling element that is also relatively positioned will automatically sit over it.',
+			},
+		},
+	},
+};
+
+function Template(args: CheckoutHeadingProps): JSX.Element {
+	return (
+		<CheckoutHeading heading={args.heading} image={args.image}>
+			{args.children}
+		</CheckoutHeading>
+	);
+}
+
+Template.args = {} as Record<string, unknown>;
+Template.parameters = {} as Record<string, unknown>;
+Template.decorators = [] as unknown[];
+
+export const Heading = Template.bind({});
+
+Heading.args = {
+	heading: 'Thank you for your\u00a0support',
+	children: (
+		<p style={{ marginRight: '48px' }}>
+			Help protect the Guardian&apos;s independence so we can keep delivering
+			quality journalism that&apos;s open for everyone around the world, not
+			behind a paywall.
+		</p>
+	),
+	image: (
+		<img src="/dev_checkout_image.png" alt="The Guardian apps and website" />
+	),
+};
+
+export const WithMenu = Template.bind({});
+
+WithMenu.args = { ...Heading.args };
+
+WithMenu.decorators = [
+	(Story: React.FC): JSX.Element => (
+		<div
+			style={{
+				backgroundColor: brand[400],
+				paddingTop: '1rem',
+			}}
+		>
+			<Container
+				topBorder={true}
+				sideBorders={true}
+				borderColor={brand[600]}
+				backgroundColor={brand[400]}
+			>
+				<div style={{ color: 'white', paddingTop: '1rem' }}>
+					[Menu component]
+				</div>
+			</Container>
+			<Story />
+		</div>
+	),
+];
+
+export const WithSiblingOverlaid = Template.bind({});
+
+WithSiblingOverlaid.args = { ...Heading.args };
+
+WithSiblingOverlaid.decorators = [
+	(Story: React.FC) => {
+		return (
+			<>
+				<Story />
+				<div
+					style={{
+						position: 'relative',
+					}}
+				>
+					<Container>
+						<Columns>
+							<Column width={[0, 0, 1 / 3]}></Column>
+							<Column>
+								<div
+									style={{
+										background: '#ffffff',
+										width: '100%',
+										maxWidth: '540px',
+										height: '600px',
+										marginTop: '120px',
```

Comment:
Nice - so the idea is that the first sibling will be overlayed, and then we can just add a top margin to push it below the heading?

Reply:
Yes, that's the idea! We might ultimately decide to move the heading out of this component and put it in a column beside the rest of the checkout, but the principle of how to overlay the content on this background remains the same.